content/en/real_user_monitoring/browser/_index.md

```diff
@@ -167,7 +175,8 @@ Add the generated code snippet to the head tag of every HTML page you want to mo
       trackResources: true,
       trackLongTasks: true,
       trackUserInteractions: true,
-    })
+    });
+   DD_RUM.startSessionReplayRecording();
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏for consistency sake, use `window.DD_RUM` instead of `DD_RUM`